packages/iTwinUI-react/stories/core/ComboBox.stories.tsx

```diff
@@ -424,8 +424,14 @@ export const CustomRenderer: Story<Partial<ComboBoxProps<string>>> = (args) => {
   }, []);
 
   const itemRenderer = React.useCallback(
```

Comment:
story controls dissapeared now.

Reply:
Any idea why they disappeared? I haven't changed anything in most of the stories.

Not that they were useful, but still.

Reply:
I think some of them returned now :/